### PR TITLE
Address Some Pending Tests

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer

File: app/controllers/snapshots_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class SnapshotsController < ApplicationController
   include Concerns::InfraLogger
 
-  before_action :authenticate_user!
+    before_action :authenticate_user!
```
Comment (root-beer) [inline app/controllers/snapshots_controller.rb]: Why did you indent here?

Comment (Joeper214) [inline app/controllers/snapshots_controller.rb]: すみません。I'll remove this indent. It was from my editor's config.

**Comment by root-beer** [comment]
LGTM! Thanks! :bow:

**Pull request merged**